lib/instrumentation/span.js

```diff
@@ -43,7 +41,7 @@ function Span (transaction, name, type, opts) {
     this._recordStackTrace()
   }
 
-  this._agent.logger.debug('start span %o', { span: this.id, parent: this.parentId, trace: this.traceId, name: name, type: type })
+  this._agent.logger.debug('start span %o', { span: this.id, parent: this.parentId, trace: this.traceId, name: this.name, type: this.type })
```

Comment:
Now that `type` doesn't contain all the same info as before, we're losing some insights with this change. Wouldn't it be worth it to include `this.subtype` and `this.action` as well? (this goes for all debug output like this across the code-base)

Reply:
fixed